test/terraform_aws_ssm_example_test.go

```diff
@@ -0,0 +1,42 @@
+package test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+)
+
+func TestTerraformAwsSsmExample(t *testing.T) {
+	t.Parallel()
+	region := "us-east-2"
+
+	terraformOptions := &terraform.Options{
+		TerraformDir: "../examples/terraform-aws-ssm-example",
+	}
+	defer terraform.Destroy(t, terraformOptions)
+
+	terraform.InitAndApply(t, terraformOptions)
+
+	instanceID := terraform.Output(t, terraformOptions, "instance_id")
+	timeout, _ := time.ParseDuration("3m")
+
+	// website::tag::1:: Wait for the instance to appear in the SSM catalog
```

Comment:
These tags are only used for examples that appear directly on the website; those are usually the "Hello, World" stye examples. Not sure we need to list this one on there?